frontend/Dockerfile

```diff
@@ -13,4 +13,4 @@ RUN npm install
 
 EXPOSE 3000
 
-CMD ["npm", "run", "local"]
\ No newline at end of file
+CMD ["npm", "run", "dev", "--", "--host", "0.0.0.0"]
```

Comment:
why is not : 
`CMD ["npm", "run", "dev"]`

Reply:
Hi @salahlalami,

Thanks for the review! The "--", "--host", "0.0.0.0" part of the command is there to ensure that the development server listens on all network interfaces inside the Docker container. Without it, the server would only be accessible from within the container itself and wouldn't respond to requests from the host machine or other containers. This change ensures that developers can access the frontend service from their local machine when the Docker container is running.

If our Docker setup or Vite's default configuration ensures the server listens on all interfaces without explicitly setting the host, we can certainly simplify the command to just "npm", "run", "dev". I included it for compatibility and accessibility, but I'm open to updating this per your guidance.